.circleci/config.yml

```diff
@@ -36,7 +36,7 @@ executors:
     docker:
       - image: metabase/ci:circleci-java-16-clj-1.10.3.929-07-27-2021-node-browsers
 
-  postgres-9-6:
+  postgres:
```

Comment:
I think keeping this name and naming the tests `be-tests-postgres-9-6-ee` actually makes more sense than `be-tests-postgres-ee` since most of the other drivers with multiple versions include the version in the test name